app/page.tsx

```diff
@@ -86,7 +86,11 @@ export default function Home() {
       ) : (
         <></>
       )}
-      <AttendeeTable attendees={attendees} setAttendees={setAttendees} />
+      {attendees.length > 0 ? (
+        <AttendeeTable attendees={attendees} setAttendees={setAttendees} />
+      ) : (
+        <></>
+      )}
```

Comment:
This check only shows the attendees button when there are attendees.. 

I'm not sure if I like this code because it reads as if the entire `AttendeeTable` is hidden and realistically it's hiding a button.